data/sql/updates/pending_db_world/rev_1676663461812301700.sql

```diff
@@ -0,0 +1,8 @@
+-- Infected Kodo Beast - add missing ability, Stampede.
+DELETE FROM `creature_template_spell` WHERE `CreatureID`=25596;
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES
+(25596, 0, 45876, 12340),
+(25596, 1, 45877, 12340);
```

Comment:
Creature_template_spell only serves for charmed creature spells. You know, the spells they can use when humanoids are charmed or demons are enslaved.

To make them cast it they need to be added to the SAI script

Reply:
And for vehicles creatures, which is this case.

Reply:
Oop, sorry lol